client/src/partials/cash/cash.js

```diff
@@ -136,6 +141,12 @@ function CashController(Cash, Cashboxes, AppCache, Currencies, Exchange, Session
   function submit(form) {
     if (form.$invalid) { return; }
 
+    // patient invoices are covered by caution
+    if (vm.slip && vm.patientBalance < 0) {
+      Notify.danger('CASH.CAUTION_COVER');
```

Comment:
We can add a confirm modal, if the client want to pay for an invoice he must accept in the confirm modal box which  display a clear message about the caution and warning.